en/diagnostics/STRDIAG010.md

```diff
@@ -0,0 +1,53 @@
+# Diagnostics Warning STRDIAG010
+
+> The Type '{0}' doesn't have a public parameterless constructor, which is needed for Serialization
```

Comment:
Full stop at the end of the sentece. Just to keep the consistency with the other STRDIAG pages intros.